cpp/include/cudf/io/experimental/hybrid_scan.hpp

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (c) 2025, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @file hybrid_scan.hpp
+ */
+
+#pragma once
+
+#include <cudf/io/parquet.hpp>
+#include <cudf/io/parquet_schema.hpp>
+#include <cudf/io/text/byte_range_info.hpp>
+#include <cudf/io/types.hpp>
+#include <cudf/types.hpp>
+#include <cudf/utilities/export.hpp>
+
+#include <memory>
+#include <utility>
+#include <vector>
+
+namespace CUDF_EXPORT cudf {
+namespace experimental::io::parquet {
```

Comment:
Just wondering if we should make this `io::experimental::parquet` instead.

Reply:
I checked the experimental JSON reader, and it was in the `cudf::io::json::experimental` namespace.
FWIW, having the `experimental` lower in the nesting (i.e. `io::parquet::experimental`) might help reduce the use of fully qualified names in the experimental code.

Reply:
Moved `experimental` to after `parquet` `cudf::io::parquet::experimental`